values.yaml

```diff
@@ -106,6 +111,33 @@ runner:
   # Arguments to pass to the `waypoint runner agent` command. Will overwrite default options.
   agentArgs: ["-vvv"]
 
+  # Address to talk to the server with. If this is not specified, this will
+  # default to the server installed with this Helm chart release. If you're
+  # installing additional runners, you must specify this since additional runners
+  # are installed using Helm releases.
+  server:
+    addr: ""
+    tls: true
+    tlsSkipVerify: true
+
+    # Cookie must be specified for runner adoption. The cookie can be retrieved
+    # using the `waypoint server cookie` CLI command or the API.
+    cookie: ""
+
+    # A secret to read the auth token from. This is optional. If this isn't
+    # set, runner adoption will be used. This only applies for runner-only
+    # installs. For server installs, the bootstrap token is used.
+    tokenSecret: ""
```

Comment:
It would be more convenient for this to be `token`, but I assume helm values files aren't safe for secrets.

It might be nice to document that the secret needs a `token` key. 

Could even document this command maybe:
`kubectl create secret generic waypoint-runner-token --from-literal=token=$(waypoint runner token)`

Reply:
Yeah I think for the documentation I'll note this.